showcase/src/app/app.module.ts

```diff
@@ -120,7 +121,7 @@ export function initRouterLog(router: UIRouter): () => void {
 export function getXsrfWaitBeforePinging(sessionService: StarkSessionService): Observable<any> {
 	let waitFor$: Observable<any> = of("production"); // no need to wait on production
 
-	if (ENV !== "production") {
+	if (ENV === "development") {
```

Comment:
But what will happen in UAT, don't wee need the logger from time to time ?

Reply:
Maybe they can just setup their applicationConfig (in uat), like the one of dev. 
In this case, my remark is useless :-)

Reply:
Actually, in the case of the showcase, we don't have any reason to have a "UAT" logic.

If we wanted to manage the different NBB environments, we could add some properties related to the environments we have in "stark-core/src/common/common/environment/environment.intf.ts" or the apps themselves could extend `StarkEnvironment` and add the needed environment, as explained here: [ENVIRONMENTS.md#how-to-add-more-properties-to-the-environment-file](https://github.com/NationalBankBelgium/stark/blob/master/docs/stark-core/ENVIRONMENTS.md#how-to-add-more-properties-to-the-environment-file)

In any case, I think this should be part of another issue. Could you please create an issue "type: discussion / RFC" with ? :blush:

Reply:
Ye sure, maybe it's useless, I don't know. 
But, it could be a small improvement.

Reply:
> @nicanac I created 2 different commits to have a commit message related to each one, like this both will be present in the CHANGELOG. What do you think ? 😊

Ok, let's keep it like that. It'll add some substance to the change log :-)